cmd/genpartial/main.go

```diff
@@ -66,151 +67,225 @@ func main() {
 	}
 }
 
-func GeneratePartial(pkg *packages.Package, structName string, out io.Writer) error {
-	root := pkg.Types.Scope().Lookup(structName)
+type Generator struct {
+	pkg     *packages.Package
+	cache   map[types.Type]ast.Expr
+	imports map[string]types.PkgName
```

Comment:
This filed is unused per staticcheck 

```
task: [ci:lint] staticcheck ./...
verifying file: charts/redpanda/ci/32-statefulset-podspec-novalues.yaml
verifying file: charts/redpanda/ci/33-advertised-ports-values.yaml
verifying file: charts/redpanda/ci/33-pod-selector-lables-novalues.yaml
verifying file: charts/redpanda/ci/34-security-contexts-novalues.yaml
verifying file: charts/redpanda/ci/34-statefulset-sidecars-novalues.yaml
verifying file: charts/redpanda/ci/35-connectors-novalues.yaml
verifying file: charts/redpanda/ci/36-single-external-address-with-template-domain-novalues.yaml
verifying file: charts/redpanda/ci/37-internal-service-changed-name-and-annotations-novalues.yaml
verifying file: charts/redpanda/ci/38-post-install-upgrade-merges-novalues.yaml
verifying file: charts/redpanda/ci/38-post-install-upgrade-no-overrides-novalues.yaml
verifying file: charts/redpanda/ci/96-audit-logging-values.yaml.tpl
verifying file: charts/redpanda/ci/97-license-key-values.yaml.tpl
verifying file: charts/redpanda/ci/98-license-secret-values.yaml
verifying file: charts/redpanda/ci/99-none-existent-config-options-with-empty-values.yaml
cmd/genpartial/main.go:73:2: field imports is unused (U1000)
```

https://github.com/redpanda-data/helm-charts/actions/runs/9666564401/job/26666371780?pr=1354#step:5:415

Reply:
Fixed!